src/vcpkg/registries.cpp

```diff
@@ -20,7 +20,7 @@ namespace
 
     using Baseline = std::map<std::string, VersionT, std::less<>>;
 
-    static const path registry_versions_dir_name = vcpkg::u8path("versions");
+    static const Path registry_versions_dir_name = "versions";
```

Comment:
```suggestion
    static constexpr const StringLiteral registry_versions_dir_name = "versions";
```
This was a path before in order to support `p / registry_versions_dir_name` without needing `u8path()`.